.env.example

```
@@ -0,0 +1,58 @@
+# ============================================================================
+SUPABASE_URL=https://your-project.supabase.co
+SUPABASE_ANON_KEY=your_anon_key
+SUPABASE_SERVICE_ROLE_KEY=your_service_role_key
+SUPABASE_DB_URL=postgresql://postgres.[project-ref]:[password]@aws-0-[region].pooler.supabase.com:6543/postgres
+
+# ============================================================================
+# Postgres / Wardrobe DB (legacy local dev)
+# ============================================================================
+POSTGRES_HOST=localhost
+POSTGRES_PORT=5433
+POSTGRES_DB=ptso_db
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_DB key should go before the POSTGRES_HOST key

(UnorderedKey)